scripts/travis/podman-test.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 set -x -e -o pipefail
 
-add-apt-repository -y ppa:projectatomic/ppa
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_18.04/Release.key -O /tmp/Release.key
+apt-key add - < /tmp/Release.key
```

Comment:
`-O-` can be used to send the output to standard output and avoid the use of temporary file:
```
wget -nv -O- https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_18.04/Release.key | apt-key add -
```

Reply:
Thanks. Changed,